public/controllers/agent/components/agents-table.js

```diff
@@ -459,7 +459,7 @@ export const AgentsTable = withErrorBoundary(
         field: 'id',
         name: 'ID',
         sortable: true,
-        show: true,
+        width: '4%',
```

Comment:
**nitpick**: This change is not present in the current `4.5` branch and I don't know if there is a reason of this change.

Maybe this is a problem resolving conflicts, because that values are present before this more recent change: https://github.com/wazuh/wazuh-kibana-app/pull/5143/files#diff-c115a3bedabddfdf963d2ea58acb86e21d3d125de731b17d93622f94c70846bb.

This is happening for all the modifications of `width` of this files. Check the another properties with the branch 4.5 if this was not intentionally changed.

It seems this was changed

Reply:
Yes, this was added when I made a rebase from 4.4-7.10.2 to 4.5. I've reverted the changes.